src/main/scala/com/strad/service/ModelsJson.scala

```diff
@@ -1,21 +1,22 @@
 package com.strad.service
 
 import cats.effect.Concurrent
-import cats.implicits._
-import io.circe.{Encoder, Decoder}
+import cats.implicits.*
+import io.circe.{Decoder, Encoder}
 import com.strad.service.domain.*
-import org.http4s._
-import org.http4s.implicits._
+import org.http4s.*
+import org.http4s.implicits.*
 import org.http4s.client.Client
 import org.http4s.client.dsl.Http4sClientDsl
-import org.http4s.circe._
-import org.http4s.Method._
+import org.http4s.circe.*
+import org.http4s.Method.*
 
 object ModelsJson:
   given Decoder[UserRequest] = Decoder.derived[UserRequest]
 
-  given [F[_] : Concurrent]: EntityDecoder[F, UserRequest] = jsonOf
+  given [F[_]: Concurrent]: EntityDecoder[F, UserRequest] = jsonOf
 
   given Encoder[UserResponse] = Encoder.AsObject.derived[UserResponse]
+  given Decoder[UserResponse] = Decoder.derived[UserResponse]
```

Comment:
Only really needed for the tests so one could argue it belongs there more than here. But in any case the routes return a UserResponse.  In the tests we need to take a UserResponse and turn it into a string.